lib/acpi/acpi.c

```diff
@@ -360,29 +364,26 @@ static int acpi_get_irq_table(char *bus_name, ACPI_PCI_ROUTING_TABLE *rt_table,
 	}
 
 	rt_buffer.Pointer = rt_table;
-	rt_buffer.Length = rt_size * sizeof(ACPI_PCI_ROUTING_TABLE);
+	rt_buffer.Length = rt_size;
```

Comment:
Why this change? I provided the justifications for passing `ARRAY_SIZE()` instead of `sizeof()` to this API in the commit message of commit 149b00f70cf8845006799edb86344a12901c53db